eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -65,6 +65,7 @@ object Transactions {
   case object OutputNotFound extends RuntimeException(s"output not found (probably trimmed)") with TxGenerationSkipped
   case object AmountBelowDustLimit extends RuntimeException(s"amount is below dust limit") with TxGenerationSkipped
 
+  case class FeeTargets(fundingBlockTarget: Int, commitmentBlockTarget: Int, mutualCloseBlockTarget: Int, claimMainBlockTarget: Int)
```

Comment:
Why is `FeeTargets` not in `fr.acinq.eclair.blockchain.fee' alongside `FeeEstimator` ?

Reply:
ACK, I moved `FeeTargets` in `FeeEstimator.scala` in commit f0649dd